datalad_container/containers_add.py

```diff
@@ -65,7 +65,8 @@ def _guess_call_fmt(ds, name, url):
     elif url.startswith('shub://') or url.startswith('docker://'):
         return 'singularity exec {img} {cmd}'
     elif url.startswith('dhub://'):
-        return op.basename(sys.executable) + ' -m datalad_container.adapters.docker run {img} {cmd}'
+        # {python} is replaced with sys.exectuable on execute
```

Comment:
```suggestion
        # {python} is replaced with sys.executable on execute
```

Reply:
Thanks, it is included in the next commit.